spec/acceptance/tests/acceptance_spec.rb

```diff
@@ -45,80 +34,46 @@
                 MANIFEST
               end
 
-    heap = if v[:elasticsearch_major_version] > 2
-             <<-MANIFEST
-               jvm_options => [
-                 '-Xms128m',
-                 '-Xmx128m',
-               ],
-             MANIFEST
-           else
-             <<-MANIFEST
-               init_defaults => {
-                 'ES_HEAP_SIZE' => '128m',
-               },
-             MANIFEST
-           end
-
     <<-MANIFEST
       api_timeout => 60,
       config => {
-        'cluster.name' => '#{v[:cluster_name]}',
-        'http.bind_host' => '0.0.0.0',
+#{es_config.map { |k, v| "        '#{k}' => '#{v}'," }.join("\n")}
       },
+      jvm_options => [
+        '-Xms128m',
+        '-Xmx128m',
+      ],
       oss => #{v[:oss]},
       #{package}
-      #{heap}
     MANIFEST
   end
 
-  context 'instance testing with' do
-    describe 'one' do
-      include_examples('basic acceptance tests', es_01)
+  context 'testing with' do
+    describe 'simple config' do
+      include_examples('basic acceptance tests', es_config)
     end
 
-    describe 'two' do
-      include_examples('basic acceptance tests', instances)
-    end
-
-    describe 'one absent' do
-      include_examples('basic acceptance tests', es_01.merge('es-02' => {}))
-    end
-
-    include_examples 'module removal', ['es-01']
+    include_examples('module removal', es_config)
   end
 
-  include_examples('template operations', es_01, v[:template])
-
-  include_examples('pipeline operations', es_01, v[:pipeline]) if semver(v[:elasticsearch_full_version]) >= semver('5.0.0')
+  include_examples('template operations', es_config, v[:template])
 
-  include_examples('plugin acceptance tests', v[:elasticsearch_plugins]) unless v[:elasticsearch_plugins].empty?
+  include_examples('pipeline operations', es_config, v[:pipeline])
 
-  # Only pre-5.x versions supported versions differing from core ES
-  if semver(v[:elasticsearch_full_version]) < semver('5.0.0')
-    include_examples(
-      'plugin upgrade acceptance tests',
-      :name => 'kopf',
-      :initial => '2.0.1',
-      :upgraded => '2.1.2',
-      :repository => 'lmenezes/elasticsearch'
-    )
-  end
+  include_examples('plugin acceptance tests', es_config, v[:elasticsearch_plugins]) unless v[:elasticsearch_plugins].empty?
```

Comment:
Looks like you've worked really hard to avoid long lines (thank you!) but this one is a beast. ;)